operators/pkg/controller/elasticsearch/migration/migrate_data_test.go

```diff
@@ -128,3 +128,42 @@ func TestMigrateData(t *testing.T) {
 		assert.Contains(t, esClient.getAndReset(), tt.want)
 	}
 }
+
+func TestIsMigratingData(t *testing.T) {
```

Comment:
A test that returns want: false would also be good to have. I know most would be covered by the more specific unit tests, but at least some litmus would be great to apply